editor/src/clj/editor/scene_visibility.clj

```diff
@@ -46,7 +46,10 @@
            {:label "Spine Scenes" :tag :spine}
            {:label "Sprites" :tag :sprite}
            {:label "Text" :tag :text}
-           {:label "Tile Maps" :tag :tilemap}]
+           {:label "Tile Maps" :tag :tilemap}
+           {:label :separator}
+           {:label "Component Guides" :tag :outline :command :toggle-component-guides :always-active true}
+           {:label "Grid" :tag :grid :always-active true}]
```

Comment:
We keep them always active, so user can on/off them independently of the rest of options.